microsoft-365/security/office-365-security/step-by-step-guides/Track and respond to emerging threats with campaigns.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Track and respond to emerging threats with campaigns
+description: Walkthrough of campaigns within Microsoft Defender for Office 365 to demonstrate how they can be used to investigate a coordinated email attack against your organization.
+search.product: 
+search.appverid: 
+ms.prod: m365-security
+ms.mktglfcycl: deploy
+ms.sitesec: library
+ms.pagetype: security
+f1.keywords: 
+  - NOCSH
+ms.author: tracyp
+author: MSFTTracyP
+ms.localizationpriority: medium
+manager: dansimp
+audience: ITPro
+ms.collection: m365-guidance-templates
+ms.topic: how-to
+ms.technology: mdo
+---
+
+# Track and respond to emerging threats with campaigns
+
+Campaigns can be used to track and respond to emerging threats because Campaigns allow you to investigate a coordinated email attack against your organization. As new threats target your organization, Microsoft Defender for Office 365 will automatically detect and correlate malicious messages. 
```

Comment:
```suggestion
Campaigns can be used to track and respond to emerging threats because campaigns allow you to investigate a coordinated email attack against your organization. As new threats target your organization, Microsoft Defender for Office 365 will automatically detect and correlate malicious messages. 
```